client/server/lib/logger/index.js

```diff
@@ -10,15 +53,20 @@ const createLogger = () => {
 		name: 'calypso',
 		streams: [
 			{
-				stream: process.stdout,
+				type: 'raw',
```

Comment:
This sends raw JSON objects instead of strings to the stream (https://github.com/trentm/node-bunyan#stream-type-raw)